benchopt/benchmark.py

```diff
@@ -304,6 +304,9 @@ def _func_cached(**kwargs):
 
     def get_config_file(self):
         "Get the location for the config file of the benchmark."
+        yml_path = self.benchmark_dir / "config.yml"
+        if yml_path.exists():
+            return yml_path
         return self.benchmark_dir / 'config.ini'
```

Comment:
I would say in this case, convert the `config.ini` to `yml`, so we can deprecate it, with a warning?